test/cypress/e2e/comment.cy.js

```diff
@@ -11,14 +11,15 @@ entryValues.forEach((entry) => {
       cy.visit('/profile')
       cy.get('[data-test="login-button"]').click().wait(3000)
 
-      cy.visit('/month').wait(3000)
       // Visits the prompt of the month
       if (entry) {
-        if (cy.visit('/2022/11/applications')) {
-          cy.visit('/2022/11/applications')
+        if (cy.visit('/2022/11/applications').wait(5000)) {
+          cy.visit('/2022/11/applications').wait(5000)
         } else {
-          cy.visit('/2022/11/-diane-m-m-daryl-mccormack-unshaken-we-ve-heard')
+          cy.visit('/2022/11/-diane-m-m-daryl-mccormack-unshaken-we-ve-heard').wait(5000)
         }
+      } else {
+        cy.visit('/month').wait(3000)
       }
```

Comment:
It's all very static here. Can't we just use `/month`?

Reply:
yeah, you are right. we can test only '`/month`'